liveRecorder/scripts/ts2mp4_build.sh

```diff
@@ -1,58 +1,69 @@
-# !/bin/bash
+#!/bin/bash
 
-if [[ $# -eq 0 ]] ; then
-    echo 'missing ffmpeg path parameter. example: ffmpeg_build ../../node_addons/FormatConverter/build/FFmpeg'
-    exit 1
-fi
-
-ffmpegLibsDir=$1
-echo ffmpeg lib dir: [$ffmpegLibsDir]
-
-currentDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-echo current dir: [$currentDir]
-
-ts2mp4ConverterDir="$currentDir/../../liveRecorder/ts_to_mp4_convertor"
-echo ts_to_mp4_convertor dir: [$ts2mp4ConverterDir]
-cd $ts2mp4ConverterDir
-
-[ -d "$ts2mp4ConverterDir/obj" ] || mkdir -p $ts2mp4ConverterDir/obj
-echo [$ts2mp4ConverterDir/obj]
-
-os_name=`uname`
-echo [$os_name]
-
-case  $os_name in
-"Darwin")
-    export OS="darwin"
-    ;;
-"Linux")
-	export OS="linux"
-   ;;
-esac
+# default bash version in Darwin is currently (May 2017), 3.2.
+# the version doesn't support {$a,,} or {$a,} to change string to lower or upper case
 
-echo os=[$OS]
-
-
-echo "********************"
-echo  Starting new build
-echo "********************"
-
-make FFMPEG_LIB_DIR=$ffmpegLibsDir
-
-echo "*******************"
-echo    Build Ended
-echo "*******************"
+APP_NAME=liveRecorder
+PREPDIR="/opt/kaltura/builds/${APP_NAME}"
+BUILD_PATH=
+WEBAPPDIR=/web/kaltura/${APP_NAME}
+BUILD_FFMPEG_PATH=${WEBAPPDIR}/ffmpeg/ffmpeg-3.0
+ROOT_PATH=.
+TARGET=ts_to_mp4_convertor
+BIN_DIR=
+BUILD_FFMPEG_PATH=
 
+if [ -e $PREPDIR ]; then
+	ROOT_PATH==${PREPDIR}/${APP_VERSION}
+	CONVERTOR_DIR=${TARGET}
+	BUILD_PATH=${ROOT_PATH}/${$CONVERTOR_DIR}
+	BUILD_FFMPEG_PATH=${WEBAPPDIR}/ffmpeg/ffmpeg-3.0
+elif [ -e ../${TARGET} ]; then
+	CONVERTOR_DIR=../${TARGET}
+	BUILD_PATH=../${TARGET}
+	BUILD_FFMPEG_PATH=../../node_addons/FormatConverter/build/FFmpeg
+fi
 
-[ -d "$currentDir/../bin/${OS}" ] || mkdir -p "$currentDir/../bin/${OS}"
+if [ -e $CONVERTOR_DIR ]; then
+	echo "CONVERTOR_DIR=${CONVERTOR_DIR}, running make to create ${TARGET}"
+	echo uname=$(uname)
+	uname=$(uname)
+	OS=darwin
```

Comment:
please remove OS dependency... since we are compling anyway
also remove bin files from repo